pybaseball/statcast_fielding.py

```diff
@@ -0,0 +1,59 @@
+import io
+from typing import Optional, Union
+
+import pandas as pd
+import requests
+
+from . import cache
+from .utils import norm_positions
+
+@cache.df_cache()
+def statcast_outs_above_average(year: int, pos: str, min_att: Union[int, str] = "q") -> pd.DataFrame:
+	# a lot of options here. need to handle all the positions. range and split years?
+	# can use startYear and endYear for multi year
+	pos = norm_positions(pos)
+	# catcher is not included in this leaderboard
+	if pos == "2":
+		raise ValueError("'C' not a valid position in this particular context!")
+	url = f"https://baseballsavant.mlb.com/leaderboard/outs_above_average?type=Fielder&year={year}&team=&range=year&min={min_att}&pos={pos}&roles=&viz=show&csv=true"
```

Comment:
Is `&viz=show` of any consequence when `csv=true`?

Reply:
Nope, works fine without it or with it set to `viz=hide`